README.md

```diff
@@ -1,5 +1,7 @@
 # neon4cast-example
 
+[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://github.com/codespaces/new?hide_repo_select=true&ref=main&repo=563603613)
```

Comment:
Maybe we should omit the button until we've debugged it entirely?  I think it should at least work if someone is a member of the org? I think still worth testing a bit more though.